app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
@kevensen @zakiva let's wait for @bzwei to comment on this because masking exceptions is misleading, redundant and not effective here (what about the other exceptions?).